src/js/control-bar/progress-control/mouse-display-bar.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file mouse-display-bar.js
```

Comment:
this isn't actually visible and is just there to align the mouse time display correctly. (ab)using the seekbar/play progress mechanism